tests/integration/src/test/java/org/apache/pulsar/tests/integration/messaging/PersistentTopicMessagingTest.java

```diff
@@ -63,4 +63,9 @@ public void testNonPartitionedTopicMessagingWithKeyShared(String serviceUrl) thr
     public void testPartitionedTopicMessagingWithKeyShared(String serviceUrl) throws Exception {
         partitionedTopicSendAndReceiveWithKeyShared(serviceUrl, true);
     }
+
+    @Test(dataProvider = "ServiceUrls")
+    public void testResetCursorCompatibility(String serviceUrl) throws Exception {
+        resetCursorCompatibility(serviceUrl, true);
```

Comment:
`resetCursorCompatibility` is a new method and you are only calling it with 'true' parameter.
is it expected ?
what about calling also` resetCursorCompatibility(serviceUrl, false);` ?